NuKeeper/Registry.cs

```diff
@@ -0,0 +1,26 @@
+﻿using NuKeeper.Configuration;
+using NuKeeper.Engine;
+using NuKeeper.Github;
+using NuKeeper.NuGet.Api;
+using SimpleInjector;
+
+namespace NuKeeper
+{
+    public static class Registry
```

Comment:
ContainerRegistration.Init or IOC.RegisterContainer seems more appropriate - Registry makes me think of operating systems and RAM